content/en/docs/collector/deployment/decentralized.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Decentralized
+description:
+  Why and how to send signals to collectors and from there to backends
+weight: 2
+---
+
+The decentralized collector deployment pattern consists of
+applications—[instrumented][instrumentation] with an OpenTelemetry SDK using
+[OpenTelemetry protocol (OTLP)][otlp]—or other collectors (using the OTLP
+exporter) that send telemetry signals to one or more [collectors][collector].
+Each client-side SDK or downstream collector is configured with a collector
+location:
+
+![Decentralized collector deployment concept](../../img/decentralized-sdk.svg)
+
+1. In the app, the SDK is configured to send OTLP data to a collector.
+1. The collector is configured to send telemetry data to one or more backends.
+
+## Example
+
+A concrete example of the decentralized collector deployment pattern could look
+as follows: you manually instrument, say, a [Java application to export
+metrics][instrument-java-metrics] using the OpenTelemetry Java SDK. In the
+context of the app, you would set the `OTEL_METRICS_EXPORTER` to `otlp` (which
+is the default value) and configure the [OTLP exporter][otlp-exporter] with the
+address of your collector, for example (in Bash or `zsh` shell):
+
+```
+export OTEL_EXPORTER_OTLP_ENDPOINT=http://collector.example.com:4318
+```
+
+The collector serving at `collector.example.com:4318` would then be configured
+like so:
+
+<!-- prettier-ignore-start -->
+{{< ot-tabs Traces Metrics Logs >}}
```

Comment:
Hmmm. Why is there an example for traces when the example scenario described above is just emitting metrics? I think the example scenario above should be changed to also include traces.

Reply:
I'm also curious why this example uses the "one collector per signal type" pattern here. Is that something we should recommend for people, or is it just happenstance that it's the example chosen here?